CONTRIBUTING.md

```diff
@@ -11,14 +12,38 @@ If you want to learn more about this project, read the [wiki](https://github.com
 
 ## Contributions
 
+Please review this [diagram](https://drive.google.com/file/d/1r7AXTrj7gtD0Xy-9BtTZR6yv7WXMPgeM/view?usp=sharing), to better understand the git workflow.
+
 We would be pleased to accept code contributions to this project.
-As we are in the alpha stage, the main priority is getting a robust blockchain up and running, with as many of the mainnet features as possible.
+As we have now released, the main priority is improving the mainnet blockchain.
 You can visit our [Trello project board](https://trello.com/b/ZuNx7sET) to get a sense of what is in the backlog.
 Generally things to the left are in progress or done. Some things go through "Coming up soon" but some will come directly out of other columns.
 Usually the things closer to the top of each column are the ones that will be worked on soonest.
 If you are interested in cryptography, math, or just like hacking in python, there are many interesting problems to work on.
 Contact any of the team members on [Keybase](https://keybase.io/team/chia_network.public), which we use as the main communication method and you can comment on any Trello card.
 
+We ask that external contributors create a fork of the `main` branch for any feature work they wish to take on.
+
+Members of the Chia organization may create feature branches from the `main` branch.
+
+In the event an emergency fix is required for the release version of Chia, members of the chia organization will create a feature branch from the current release branch `1.0.0`
+
+## Branching Strategy
+
+[Branching Strategy Diagram](https://drive.google.com/file/d/1NF2zdQOfCGlcRu6qKMlV_NlO4uqnIvoE/view?usp=sharing)
+
+1. All changes go into the main branch.
+2. Main is stable at all times, all tests pass.
+3. Features (with tests) are developed and fully tested on feature branches, and reviewed before landing in main.
+4. The public testnet is running the current release branch i.e `1.0.0`.
+5. The `main` branch will have a long running `beta testnet` to allow previewing of changes.
+6. Pull Request events may require a `beta testnet` review environment, at the moment this is at the discretion of the reviewer.
+7. Hotfixes land in the release branch they fix, and all later versions. (this will be achieved by regularly merging from 1.0.x -> main).
```

Comment:
Very slight semantic change: You might have a case where you want the functionality to go from a release branch to main, but not via "git merge".

No big deal.

Reply:
merge is used colloquially here to infer "interaction that brings changes from one branch to another" this will often likely take the form of git cherry picks if were being honest. Do you think there is value in being more precise here? I was aiming for "broad audience" but I might have aimed too wide.